frontend/src/views/SignUpView.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <h1>S'Inscrire</h1>
+    <form
+      @submit.prevent="signup"
+      style="border: solid 1px black; width: 25vw; min-width: 300px"
+      class="mx-auto p-3"
+    >
+      <div class="bg-danger text-white p-1 mx-4 rounded" v-if="erreurs.length">
+        <span v-for="erreur in erreurs" :key="erreur">{{ erreur }}</span>
+      </div>
+      <input
+        class="text-center p-1 my-3 rounded"
+        style="background-color: #e3e3e3"
+        v-model="courriel"
+        placeholder="Courriel"
+      />
+      <div style="width: 100%"></div>
+      <input
+        class="text-center p-1 rounded"
+        style="background-color: #e3e3e3"
+        v-model="nom"
+        placeholder="Nom"
+      />
+      <div style="width: 100%"></div>
```

Comment:
Same here for the style attributes, you can even use the simple 'w-100' class provided by bootstrap

Reply:
Ah, I hadn't thought of that.